docs/Clojure.md

```diff
@@ -0,0 +1,84 @@
+# How to use Minio SDK for java with Clojure
+
+In this receipe we will learn how to use `minio-java` with Clojure.
+
+## Prerequisites
+Install [`Clojure`](https://clojure.org/community/downloads) and [`Leiningen`](https://leiningen.org/).
+
+Check by running -
+```
+lein --help
+```
+
+## Clojure Project
+Following are the steps involved foe using minio-java with Clojure : 
+
+### 1. Setup
+Create a new Clojure project using Leiningen.
+```
+lein new <templateName> <projectName>
+```
+
+```
```

Comment:
This feels incomplete, please add a line here like `For example` or similar

Reply:
Okay.